### DISCUSSION TITLE
Can we use fluentbit 3.0 ?

### DISCUSSION THREAD

**Discussion by rufdoSICKAG** [fluent/fluent-operator#1141] [General]
Hi,
what are the plans for the new fluentbit 3.0?
AFAIS fluent-operator still uses v2.2.2.
Can I just change my values.yaml to use a 3.0.x image?

**Comment by wenchajun** [comment]
We currently cannot use the official image version because we have added plugins to the official image for configuration loading. The 3.0 version has updated many features, and we need to adapt and adjust. We will release the 3.x version of the image in the coming months
